projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.spec.ts

```diff
@@ -303,10 +303,6 @@ describe('igxExpansionPanel', () => {
             const ariaExpanded = collapsed ? 'false' : 'true';
             expect(panelHeader.querySelector('div [role = \'button\']').getAttribute('aria-expanded')).toMatch(ariaExpanded);
             expect(panelHeader.classList.contains(CSS_CLASS_HEADER_EXPANDED)).toEqual(!collapsed);
-            if (button.children.length > 1) {
-                const iconName = collapsed ? 'expand_more' : 'expand_less';
-                expect(button.getAttribute('ng-reflect-icon-name')).toMatch(iconName);
```

Comment:
While the grid reflect checks are silly, this one doesn't seem to be somewhat reasonable check the correct icon is shown and there doesn't seem to be other checks. Perhaps it should be re-done with fixture query with the icon component and directly check it's set collection + name.